_javascripts/src/index.js

```diff
@@ -0,0 +1,11 @@
+import 'core-js/stable'
```

Comment:
Do we need this? Doesn't Babel do that for us?

Reply:
Apparently not. I've just tried and we get a bunch of errors in IE8